doc/source/serve/index.md

```diff
@@ -121,9 +124,85 @@ Because it's built on top of Ray, you can run it anywhere Ray can: on your lapto
+
+:::
+
+
+## How does Serve compare to ...
+
+:::{dropdown} TFServing, TorchServe, ONNXRuntime
+:animate: fade-in-slide-down
+
+Ray Serve is *framework agnostic*, so you can use any Python framework and libraries.
+We believe data scientists are not bounded a particular machine learning framework.
+They use the best tool available for the job.
+
+Compared to these framework-specific solutions, Ray Serve doesn't perform any optimizations to make your ML model run faster. However, you can still optimize the models yourself
+and run them in Ray Serve. for example, you can run a model compiled by
+[PyTorch JIT](https://pytorch.org/docs/stable/jit.html) or [ONNXRuntime](https://onnxruntime.ai/).
+:::
+
+:::{dropdown} AWS SageMaker, Azure ML, Google Vertex AI
+:animate: fade-in-slide-down
+
+Ray Serve brings the scalability and parallelism of these hosted offerings to
+your own infrastructure. You can use our [cluster launcher](cluster-cloud)
+to deploy Ray Serve to all major public clouds, K8s, as well as on bare-metal, on-premise machines.
+
+Compared to these offerings, Ray Serve lacks a unified user interface and functionality to
+let you manage the lifecycle of the models, visualize it's performance, etc. Ray
+Serve primarily focuses on model serving and provides the primitives for you to
+build your own ML platform on top.
+:::
+
+:::{dropdown} Seldon, KServe, Cortex
+:animate: fade-in-slide-down
+
+You can develop Ray Serve on your laptop, deploy it on a dev box, and scale it out
+to multiple machines or a K8s cluster with minimal or no changes to code. It's a lot
+easier to get started with when you don't need to provision and manage a K8s cluster.
+When it's time to deploy, you can use our [Kubernetes Operator](https://ray-project.github.io/kuberay/guidance/rayservice/)
+to transparently put your Ray Serve application in K8s.
+:::
+
+:::{dropdown} BentoML, Comet.ml, MLflow
+:animate: fade-in-slide-down
+
+Ray Serve is a special purpose distributed model server built for large scale applications.
+This means we can work with any model packaging and registry format.
+Many of the tools are focused on serving one ML framework-specific type of a model. By contrast, Ray Serve is framework-agnostic and allows you to build an end-to-end machine learning application with its unique model composition API and advanced autoscaling capabilities.
+:::
+
+We truly believe Serve is unique as it gives you end to end control
+over the API while delivering scalability and high performance. To achieve
+Serve's feature offerings, you often need to glue together multiple
+frameworks like Tensorflow Serving, SageMaker, or even roll your own
+batching server.
```

Comment:
```suggestion
Many of these tools are focused on serving models for one specific ML framework. In contrast, Ray Serve is framework-agnostic and focuses on large-scale model serving in general.
As such, Ray Serve works with any model packaging and registry format.
Ray Serve also provides key features for building end-to-end machine learning applications, including model composition and autoscaling.
:::

We truly believe Serve is unique as it gives you end-to-end control
over the API while delivering scalability and high performance. To achieve
Serve's feature offerings with other tools, you would need to glue together multiple
frameworks like Tensorflow Serving and SageMaker, or even roll your own
batching server.
```

What tools are "the tools" referring to? I'm going to assume it's the ones that are listed at the top. Reordered to make this clearer.

Also don't know what "the API" is referring to, or why it's significant to have end-to-end control over it. Serve's API?

What's a "batching server"?